packages/core/src/fhir-to-cda/cda-templates/components/__tests__/allergies.test.ts

```diff
@@ -0,0 +1,62 @@
+import { faker } from "@faker-js/faker";
+import _ from "lodash";
+import { Bundle, AllergyIntolerance } from "@medplum/fhirtypes";
+import path from "path";
+import { removeEmptyFields } from "../../clinical-document/clinical-document";
+import { xmlBuilder } from "../../clinical-document/shared";
+import { buildAllergies } from "../allergies";
+import { allergyFood, allergyMedication } from "./allergy-examples";
+import { makeAllergy } from "./make-allergy";
+import { createEmptyBundle, getXmlContentFromFile } from "./shared";
+
+let allergyId: string;
+let bundle: Bundle;
+let allergy: AllergyIntolerance;
+
+beforeEach(() => {
+  allergyId = faker.string.uuid();
+  allergy = makeAllergy({
+    id: allergyId,
+    ...allergyMedication,
+  });
+
+  bundle = createEmptyBundle();
+});
+
+describe("buildAllergies", () => {
+  it("correctly maps a single AllergyIntolerance", () => {
+    bundle.entry?.push({ resource: { ...allergy, note: [] } });
```

Comment:
why override with empty note?

Reply:
kinda just testing more variations in a smaller number of tests. In this scenario, we're not including the optional note in to the single AllergyIntolerance